docs/readmes/nms/alert.md

```diff
@@ -0,0 +1,322 @@
+|mme_new_association	|New SCTP association	|AGW	|No	|MME	|
+|ue_context_release_command_timer_expired	|UE context release command timer expired	|AGW	|No	|MME	|
+|enb_sctp_shutdown_ue_clean_up_timer_expired	|SCTP shutdown UE clean up timer expired	|AGW	|No	|MME	|
+|ue_context_release_req	|UE context release request	|AGW	|No	|MME	|
+|s1ap_error_ind_rcvd	|S1AP error indication received	|AGW	|No	|MME	|
+|s1_reset_from_enb	|S1 reset from eNB	|AGW	|No	|MME	|
+|nas_non_delivery_indication_received	|NAS non delivery indication received	|AGW	|No	|MME	|
+|spgw_create_session	|SPGW create session success	|AGW	|No	|MME	|
+|ue_pdn_connection	|UE PDN connection	|AGW	|No	|MME	|
+|ue_pdn_connectivity_req	|UE PDN connectivity request	|AGW	|No	|MME	|
+|	|	|	|	|	|
+|ue_reported_usage / up	|Reported TX traffic for subscriber / session in bytes	|AGW	|Yes	|sessiond	|
+|ue_reported_usage / down	|Reported RX traffic for subscriber / session in bytes	|AGW	|Yes	|sessiond	|
+|ue_dropped_usage / up	|Reported dropped TX traffic for subscriber / session in bytes	|AGW	|No	|sessiond	|
+|ue_dropped_usage / down	|Reported dropped RX traffic for subscriber / session in bytes	|AGW	|No	|sessiond	|
+
+More up to date information might be available from the “Metrics Explorer” in the metrics component.
+![alerts4](assets/nms/userguide/alerts/alerts4.png)
+**Custom Alert Configuration**
+
+An alert configuration consists of
+
+* Name/Description of the alert
+* Alert Definition
+* Alert receiver to be notified when the alert is fired
+* Additional labels which can be added to provide more information about the alert.
+
+Alert definition is the most interesting attribute in the configuration. It consists of a metric expression(based on prometheus promQL expression) and a duration attribute which specifies the time for the expression to be true, following which the alert is fired.
+![alerts5](assets/nms/userguide/alerts/alerts4.png)![alerts6](assets/nms/userguide/alerts/alerts4.png)
+
+Following examples shows how we can use the above mentioned metrics and add custom alerts based on them.
+
+**eNB Down Alert:**
+This alert fire if eNB Rf Tx is down in any of the gateways in your network.
+
+Expression:
+
+```
+sum by(gatewayID) (enodeb_rf_tx_enabled{networkID="<your network ID>"} < 1)
+```
+
+Duration:
+10 minutes
+
+**No Connected eNB Alert:**
+This alert fire if the connected eNB count falls to ‘0’ for any of the gateways in your network.
+
+Simple: Select “enb_connected” metric from the dropdown and construct the if statement as **“if enb_connected < 1”**
+
+Advanced Expression:
+
+```
+(enb_connected{networkID="<your network ID>"}) < 1
+```
+
+Duration:
+5 minutes
+
+**Free Memory is < 10 Alert:**
+This alert fire if the Free Memory of any of the gateways in your network is less than 10%
+
+Advanced Expression:
+
+```
+((1 - mem_available{networkID="<your network ID>"} / mem_total{networkID=~"<your network ID>"}) * 100) > 90
```

Comment:
This is just a suggestion on the alert itself. As it is it's fine for an example. You should do an `avg_over_time(metric[5m]) > 90` to avoid randomness preventing this from firing. The way your expression is written if there's a single datapoint at 89 during those 15 minutes then it won't fire.